### PR TITLE
Add custom domains docs

### PR DESCRIPTION
**Pull request by LukeSettle**
Add docs for managing custom domains

### CODE REVIEW

**Review by jacobherrington — APPROVED** [review]
@chadwickcheney Should probably take a look at these!

File: _includes/content/domains/link_dns_target.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ol>
+  <li>Log in to the account you have with your current domain provider.</li>
+  <li>Find your DNS settings for the domain you want to use.</li>
+  <li>Create a CNAME record using the DNS target value provided below</li>
```
Comment (juliannatetreault) [inline _includes/content/domains/link_dns_target.html]: This `li` is missing a colon or a period at the end of it.

File: domains/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - title: Getting to the Domains Homepage
+    steps:
+
+    - instruction: Login to the Dashboard
+      image: '/assets/images/screenshots/draft2/login.png'
+
+    - instruction: Click the Side Bar menu button to the left of the page
+      image: /assets/images/mockdown/draft2/domains/open-the-sidebar.png
+
+    - instruction: Locate the Settings dropdown menu inside the side bar menu to the left of the page
```
Comment (juliannatetreault) [inline domains/index.md]: Tiny tweak, that's a non-issue: On line 16, "Side Bar" is capitalized, but on line 19 it is not. I would suggest making both (lines 16 and 19) capitalized or lower case to keep the docs consistent.

File: domains/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    - instruction: Click the Stores link in the drop down
+      image: /assets/images/mockdown/draft2/domains/sidebar-expanded.png
+
+    - instruction: This takes you to a list of your stores
+      notes:
+      - header: Click the "Domains" button for the store you would like to add your custom domain to
+      image: /assets/images/mockdown/draft2/domains/stores-index.png
+
+    - instruction: This is the Domains Homepage. Here you will find all the information you need to maintain the domains tied to your store.
```
Comment (juliannatetreault) [inline domains/index.md]: This is the only `instruction` that has a period at the end, so I would delete it to keep things consistent.

**Review by juliannatetreault — APPROVED** [review]
Just a couple of non-blocking comments, but overall all looks good!

**Pull request merged**